Makefile

```diff
@@ -40,7 +40,7 @@ include common/packages.mk
 
 GHA_MATRIX ?= minimal
 ifeq ($(GHA_MATRIX),minimal)
-	GHA_RELEASES := debian10 debian11 debian12 ubuntu2004 ubuntu2204 ubuntu2304 centos7 centos9 oraclelinux7 fedora38 fedora39 static
+	GHA_RELEASES := debian10 debian11 debian12 ubuntu2004 ubuntu2204 ubuntu2304 ubuntu2310 centos7 centos9 oraclelinux7 fedora38 fedora39 static
```

Comment:
Perhaps `ubuntu2310` should be _replacing_ `ubuntu2304`?

Reply:
It's still in our official pipeline https://github.com/docker/docker-ce-packaging/blob/d76f75ac88412dd1d0e470f56f3ae7dd546d07d8/Jenkinsfile#L20 and EOL next year so I think we should keep it for now (cc @thaJeztah @neersighted)